index.html

```diff
@@ -42,6 +47,20 @@
       </div>
     </header>
 
+    <div class="hamburguer">
+      <button id="btn-close" type="button" class="btn-close" aria-label="Close"></button>
+      <ul>
+        <li>
+          <a class="hamburguer-link" href="#Portoflio"><h3>Portfolio</h3></a>
+        </li>
+        <li>
+          <a class="hamburguer-link" href="#About"><h3>About</h3></a>
+        </li>
+        <li>
+          <a class="hamburguer-link" href="#Contact"><h3>Contact</h3></a>
+        </li>
+      </ul>
+    </div>
```

Comment:
- [ ] You did a great job working on your mobile menu! 💯 However, there's still some room to improve the design. One of the main goals of this project is that you show your ability to follow as closely as possible a given design, so please try to stick as much as possible to the Figma template! 🎨 Please consider increasing the font-size and the height of the menu to fill the entire screen. 🤓 🛠️ 


|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/210391734-2b006572-371e-480b-b05a-9c6cbdbb2343.png) | ![image](https://user-images.githubusercontent.com/77069011/210391374-85fcab9c-f976-4094-a6be-aa40f2fa47c7.png) |